tests/functional/addons-install-test.bats

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bats
+
+# Check if TERMINUS_SITE is defined, otherwise define it as terminus-addons-installer-plugin.
+if [ -z "$TERMINUS_SITE" ]; then
+  # Looks like this is a local run. We'll create the localtests multidev ro test this.
+  TERMINUS_SITE=terminus-addons-installer-plugin
+
+  # Check if the localtests environment exists already, otherwise create it.
```

Comment:
We good with "localtests" being a single env so no more than one eng can test locally at a time? what about checking an env variable (but could still default to localtests)